castervoice/lib/settings.py

```diff
@@ -481,7 +481,10 @@ def initialize():
     # calculate prerequisites
     SYSTEM_INFORMATION = _get_platform_information()
     _BASE_PATH = str(Path(__file__).resolve().parent.parent)
-    _USER_DIR = user_data_dir(appname="caster", appauthor=False)
+    if os.getenv("CASTER_USER_DIR") is not None:
+        _USER_DIR = os.getenv("CASTER_USER_DIR")
```

Comment:
Does `CASTER_USER_DIR` create a permanent entry or is it only temporary for while casters running?

Reply:
os.getenv does not create an entry. It just looks for and reads an existing environment variable named `CASTER_USER_DIR`.

That environment variable can exist temporarily or be set permanently.